src/cloud_what/providers/azure.py

```diff
@@ -107,12 +107,16 @@ def is_likely_running_on_cloud(self):
         # We know that Azure uses only HyperV
         if 'virt.host_type' in self.hw_info:
             if 'hyperv' in self.hw_info['virt.host_type']:
-                probability += 0.3
+                probability += 0.2
 
         if 'dmi.chassis.asset_tag' in self.hw_info and \
                 self.hw_info['dmi.chassis.asset_tag'] == '7783-7084-3265-9085-8269-3286-77':
             probability += 0.3
 
+        # The waagent log exists and contains the string "Azure Linux Agent"
```

Comment:
I'm not against the idea, but isn't it possible that some public/private cloud provider use this client too. It is open source and there is some chance that could be used outside of Azure. For this reason I would not increase the probability so much.

Reply:
Yes, but I don't think that some other cloud provider uses this agent. If it is the case, then we can adjust it later on, too.
The reason why I increased the "probability" so much is, that then the detection would also work if dmi-decode data doesn't exist.